app/assets/javascripts/app/core/models/discussion.js

```diff
@@ -5,63 +5,63 @@
     .module('council.core')
     .factory('Discussion', Discussion);
 
-  function Discussion(DS, $http, $q) {
+  function Discussion(DiscussionComment, User, DS, $http, $q) {
```

Comment:
I tried this, I cannot remove the `DiscussionComment`, I think it's because the json is nested.